static/js/apps/explore/app.tsx

```diff
@@ -413,6 +418,9 @@ const fetchDetectAndFufillData = async (
   if (client) {
     argsMap.set(URL_HASH_PARAMS.CLIENT, client);
   }
+  if (defaultPlace) {
```

Comment:
Is this necessary? won't this be handled by the search context?

Reply:
sorry, what do you mean by the search context? Do you mean to just always set default place in the args map instead of having it in a conditional?

Reply:
i was thinking-do we have to set this url parameter at all for subsequent searches? if the user doesn't specify a place, will the user still get the same default place (only out of the context history instead of the default_place arg)

Reply:
if there is a place in the context history, subsequent searches will just use that. But this call is used in both the first search and later searches, so we would still need to set default_place?

Reply:
That makes sense!